### PR TITLE
jax.numpy.rint

### PR DESCRIPTION
**Pull request by ZnMind**

### CODE REVIEW

**Comment by ZnMind** [comment]
Close #14822

**Comment by satakshigarg** [comment]
Hey @ZnMind , thanks a lot for the PR.

However, there are some lint and formatting checks that are failing. I guess you haven't installed pre-commit yet. I suggest you [set up](https://lets-unify.ai/ivy/overview/contributing/setting_up.html#pre-commit) pre-commit in your local environment and [run](https://lets-unify.ai/ivy/overview/deep_dive/formatting.html#pre-commit-hook) these lint and formatting checks.
Besides, please try running your tests locally, where pytest will show failed tests on your local system itself. Kindly always make sure that you run all your tests locally and ensure that they pass for ALL backends because otherwise the CI tests will fail when you submit a PR.
When you've done all these changes and verified that the tests are passing, feel free to ping me for a review.

Good luck!

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.
